internal/nostr/models.go

```diff
@@ -14,26 +14,30 @@ const (
 	NIP_47_RESPONSE_KIND   = 23195
 
 	// state of request event
-	REQUEST_EVENT_PUBLISH_CONFIRMED   = "confirmed"
-	REQUEST_EVENT_PUBLISH_FAILED      = "failed"
+	REQUEST_EVENT_PUBLISH_CONFIRMED = "CONFIRMED"
```

Comment:
can you add a notification kind above for 23196?

Reply:
Done